components/serializer/encoders.rst

```diff
@@ -0,0 +1,105 @@
+.. index::
+   single: Serializer, Encoders
+
+Encoders
+========
+
+Encoders basically turn **arrays** into **formats** and vice versa.
+They implement :class:`Symfony\\Component\\Serializer\\Encoder\\EncoderInterface` for encoding (array to format) and :class:`Symfony\\Component\\Serializer\\Encoder\\DecoderInterface` for decoding (format to array).
```

Comment:
Please wrap lines after the first word that crosses the 72nd character (seem below).